tests/ha/test_control_pod_restart.py

```diff
@@ -729,3 +731,310 @@ def test_mpu_after_ctrl_pod_failover(self):
                     "and downloaded 5GB object")
 
         LOGGER.info("ENDED: Verify multipart upload before and after control pod failover")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-40387")
+    @CTFailOn(error_handler)
+    def test_ctrl_pod_failover_loop(self):
+        """
+        Verify control pod failover in loop
+        """
+        LOGGER.info("STARTED: Verify control pod failover in loop")
+
+        total_buckets = HA_CFG["s3_bucket_data"]["no_bck_background_deletes"]
+        event = threading.Event()
+        output = Queue()
+
+        LOGGER.info("Step 1: Create IAM user and perform WRITEs-READs-Verify with variable object"
+                    " sizes.")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.test_prefix = 'test-40387'
+        self.s3_clean = users
+        uids = list(users.keys())
+        access_key = list(users.values())[0]["accesskey"]
+        secret_key = list(users.values())[0]["secretkey"]
+        s3_test_obj = S3TestLib(access_key=access_key, secret_key=secret_key,
+                                endpoint_url=S3_CFG["s3_url"])
+
+        LOGGER.info("Create %s buckets and put variable size objects and perform read-DI check.",
+                    total_buckets)
+        args = {'test_prefix': self.test_prefix, 'test_dir_path': self.test_dir_path,
+                'skipdel': True, 'bkts_to_wr': total_buckets, 'di_check': True, 'output': output}
+
+        self.ha_obj.put_get_delete(event, s3_test_obj, **args)
+        wr_resp = tuple()
+        while len(wr_resp) != 3:
+            wr_resp = output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        s3_data = wr_resp[0]  # Contains s3 data for passed buckets
```

Comment:
Contains s3 or IAM?

Reply:
s3 data as in bucket names, object names etc. It doesn't mean s3 account data.